src/App.jsx

```diff
@@ -1,13 +1,13 @@
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 
+import { HistoryButton } from "./sidePanel-history-button.jsx";
```

Comment:
다음과 같이 컴포넌트 영역 별로 불러오면 좋을 것 같습니다 :)

```suggestion
import { HistorySection } from "./components/history-section/index.jsx";
import { SearchSection } from "./components/search-section/index.jsx";
import { LatelyHistoryGroup } from "./components/lately-history-group/index.jsx";
```